InnerEye/ML/configs/other/fastmri_varnet.py

```diff
@@ -32,8 +32,8 @@ class VarNetWithImageLogging(VarNetModule):
     """
 
     def log_image(self, name: str, image: torch.Tensor) -> None:
-        experiments = self.logger.experiment if isinstance(self.logger.experiment, list) \
-            else [self.logger.experiment]
+        experiments = self.loggers[0].experiment if isinstance(self.loggers[0].experiment, list) \
```

Comment:
I don't get this change. Does the `logger` field no longer exist in new PL?

Reply:
Yep precisely!

Reply:
Strange 🤔 You should be able to use logger property that return the first logger in self.loggers
<img width="429" alt="image" src="https://user-images.githubusercontent.com/37396332/190124568-363504a1-6fd2-448f-86f1-1b809b9b6ddf.png">
This is defined in LightningModule which I believe `VarNetWithImageLogging` inherits from